raiden/tests/scenarios/mfee2_proportional_fees.yaml

```diff
@@ -0,0 +1,79 @@
+version: 2
+
+settings:
+  gas_price: "fast"
+  chain: any
+  services:
+    pfs:
+      url: https://pfs-goerli-with-fee.services-dev.raiden.network
+    udc:
+      enable: true
+      token:
+        deposit: true
+
+token:
+  address: "0x62083c80353Df771426D209eF578619EE68D5C7A"
+  balance_min: 1_000_000
```

Comment:
this will also need `balance_fund`, right?

Reply:
Ah yes. Forgot that I was using a local version of the SP that had another fix for the problem. Will update it.